bin/gftools-check-sandbox-family.py

```diff
@@ -0,0 +1,85 @@
+"""Make a diff/gif image of a site by swapping the GF production families
+with same families hosted on the sandbox server.
+
+
+Usage:
+gftools check-sandbox-family https://www.somesite.com
+"""
+import os
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options 
+from PIL import Image, ImageDraw, ImageFont
+import argparse
+from urllib.parse import urlsplit
+from io import BytesIO
+import time
+
+
+JS = """
+links = document.getElementsByTagName('link');
+for (i in links) {
+    if (typeof(links[i].href) != 'undefined') {
+         if (links[i].href.indexOf("fonts.googleapis.com") > -1)             
+             links[i].href = links[i].href.replace("fonts.googleapis", "fonts.sandbox.google");
+    }
+}
+"""
+WIDTH = 1024
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("url")
+    args = parser.parse_args()
+
+    chrome_options = Options()
+    chrome_options.add_argument("--headless")
+
+    with webdriver.Chrome(options=chrome_options) as driver:
+        driver.get(args.url)
+        required_height = driver.execute_script('return document.body.parentNode.scrollHeight')
+        driver.set_window_size(WIDTH, required_height)
+
+        try:
+            families_in_use = driver.find_elements_by_xpath(
+                    '//link[contains(@href, "fonts.googleapis.com/css")]'
+            )
+            for family in families_in_use:
+                print("Changing GF url %s to %s" % (
+                    family.get_attribute("href"), family.get_attribute("href").replace(
+                        "fonts.googleapis.com", "fonts.sandbox.google.com")
```

Comment:
`fonts.sandbox.google.com` can be replaced with any GET request accessible URL to a CSS file?

Reply:
Ah! I see it up here:

https://github.com/googlefonts/gftools/pull/116/files#diff-2331a623dfd3e84252439986a266425cR23

I will take a closer look at this.  Thanks Marc!

Reply:
Making a more generic swapper is fine, but I think we need a script specifically for our sandbox.